actions.py

```diff
@@ -161,7 +162,6 @@ def calculate_emissions(self, tracker, unit="short_tons"):
         departure_iata = tracker.get_slot("iata_departure")
         destination_iata = tracker.get_slot("iata_destination")
         flight_class = tracker.get_slot("travel_flight_class")
-        stopover_iata = tracker.get_slot("iata_stopover")
```

Comment:
Have we removed the whole stopover thing altogether?

Reply:
I removed it for this update, yes. Chris had commented it out, but I don't like to keep commented-out code (that's why we have git 😅).

There were a couple of other features where it wasn't clear if they were finished and how well they worked, like co-reference resolution of cities, correcting cities, and stopover. Maybe I should make issues to remind us that these existed? What do you think?

Reply:
Yes, I'd say issues for any of these will be a good idea so we have a record of them on the books.